php/blocks/controls/class-image.php

```diff
@@ -63,8 +63,10 @@ public function register_settings() {
 	 * @return string|int $value The value to be made available or echoed on the front-end template, possibly 0 if none found.
 	 */
 	public function validate( $value, $echo ) {
+		$image_id = intval( $value );
```

Comment:
Hi @lukecarbis, this looks good. I haven't tested it, but if you're happy, I'm happy.

Reply:
Good idea to set up Travis, it's good that it caught this.